### PR TITLE
[bitnami/airflow] Allow using custom secret key with external database.

### PR DESCRIPTION
**Pull request by sgrzemski**
**Description of the change**

I am using ExternalSecrets and all my secret keys are capitalized. I would like to be able to specify the key inside a secret to integrate airflow with my current setup. I've added a configurable field to determine the actual key name and use this chart in multiple configurations.  `postgresql-password` is still a default to stay compliant with your postgres chart.

**Benefits**

An ability of specifying the secret key that contains password. Make this chart compatibie with ExternalSecrets.

**Possible drawbacks**

None.

**Applicable issues**

  - fixes #

**Additional information**

Tested with the following values (but not only):
```
executor: 'KubernetesExecutor'
loadExamples: false
redis:
  enabled: false

postgresql:
  enabled: true
  existingSecret: some-hacky-password
```
```
executor: 'KubernetesExecutor'
loadExamples: false
redis:
  enabled: false

postgresql:
  enabled: false
  existingSecret: some-hacky-password

externalDatabase:
  existingSecret: super-secret
  existingSecretPasswordKey: CHANGE_ME
```
and got this:
```
            - name: AIRFLOW_DATABASE_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: super-secret
                  key: CHANGE_ME
```

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/airflow/templates/_helpers.tpl [hunk 1]
```diff
@@ -220,6 +220,25 @@ Add environment variables to configure database values
 {{- ternary .Values.postgresql.postgresqlDatabase .Values.externalDatabase.database .Values.postgresql.enabled | quote -}}
 {{- end -}}
 
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "airflow.database.existingsecret.key" -}}
+{{- if (.Values.postgresql.enabled) -}}
```
Comment (javsalgar) [inline bitnami/airflow/templates/_helpers.tpl:227]: You don't need the brackets. Apply to the rest of the `if` statements

**Review by javsalgar — COMMENTED** [review]
Thanks for the PR! Minor change requested

File: bitnami/airflow/templates/_helpers.tpl (hunk 1 above)
Comment (sgrzemski) [reply bitnami/airflow/templates/_helpers.tpl]: Done, thank you!

**Review by javsalgar — APPROVED** [review]
LGTM

**Pull request merged**